edx_proctoring/backends/tests/test_backend.py

```diff
@@ -149,3 +150,23 @@ def test_null_provider(self):
         self.assertIsNone(provider.get_software_download_url())
         self.assertIsNone(provider.on_review_callback(None))
         self.assertIsNone(provider.on_review_saved(None))
+
+    def test_mock_provider(self):
+        """
+        Test that the mock backend provider does what we expect it to do.
+        """
+        provider = MockBackendProvider()
+        attempt_code = "test_code"
+        self.assertEqual(
+            attempt_code,
+            provider.register_exam_attempt(None, {'attempt_code': attempt_code})
+        )
+
+        self.assertEqual(
```

Comment:
Shouldn't this also assert that `start_exam_callback` is eventually called, or is that too difficult?